src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands/AddAzureRmServiceFabricNode.cs

```diff
@@ -21,6 +21,38 @@ namespace Microsoft.Azure.Commands.ServiceFabric.Commands
     [Cmdlet(VerbsCommon.Add, CmdletNoun.AzureRmServiceFabricNode, SupportsShouldProcess = true), OutputType(typeof(PSCluster))]
     public class AddAzureRmServiceFabricNode : UpdateAzureRmServiceFabricNodeBase
     {
+        /// <summary>
+        /// Resource group name
+        /// </summary>
+        [Parameter(Mandatory = true, Position = 0, ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specify the name of the resource group.")]
+        [ValidateNotNullOrEmpty()]
+        public override string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, Position = 1, ValueFromPipelineByPropertyName = true,
+                   HelpMessage = "Specify the name of the cluster")]
+        [ValidateNotNullOrEmpty()]
+        [Alias("ClusterName")]
+        public override string Name { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipeline = true,
+           HelpMessage = "Node type name")]
+        public override string NodeType { get; set; }
+
+        protected override int Number
+        {
+            get { return this.NumberOfNodesToAdd; }
+        }
+
+        [Parameter(Mandatory = true, ValueFromPipeline = true,
+          HelpMessage = "The number of nodes to add")]
+        [ValidateRange(1, 2147483647)]
+        [Alias("Number")]
+        public int NumberOfNodesToAdd
```

Comment:
I am not positive that this alias to 'Number' works.  Why not just have one parameter called number?